readthedocs/subscriptions/tasks.py

```diff
@@ -62,7 +62,9 @@ def daily_email():
 @app.task(queue="web")
 def disable_organization_expired_trials():
     """Daily task to disable organization with expired Trial Plans."""
-    queryset = Organization.objects.subscription_trial_plan_ended()
+    queryset = Organization.objects.disable_soon(
+        days=30, exact=True
+    ).subscription_trial_plan_ended()
```

Comment:
Why do we need both here, `disable_soon` and `subscription_trial_plan_ended`?

Reply:
We are automatically disabling expired trial subscriptions only, we don't do that automatically for normal subscriptions (I think we should start doing it automatically as well)